events/wc_events/wc_central_kalimdor_events.txt

```diff
@@ -0,0 +1,3065 @@
+			}
+			random_neighboring_and_across_water_top_liege_realm_owner = {
+				limit = { NOT = { this = root } }
+				save_scope_as = kodo_owner
+			}
+		}
+		create_character = {
+			template = bandit_character
+			location = root.location
+			culture = root.location.culture
+			faith = root.location.faith
+			gender = male
+			save_scope_as = shoeless_bandit
+		}
+	}
+	
+	#Lead Attack
+	option = {
+		name = cen_kal.1.a
+		if = { limit = { has_trait = craven } add_stress = 25 }
+		if = { limit = { has_trait = brave } add_stress = -25 }
+		add_prestige = 30
+		trigger_event = { id = cen_kal.1015 days = { 2 5 } }
+		custom_tooltip = cen_kal.1.a.tt
+		ai_chance = { 
+			base = 50
+			modifier = {
+				add = 35
+				has_trait = brave
+			}
+			modifier = {
+				add = 10
+				NOT = { has_trait = wounded_1 }
+			}
+			modifier = {
+				add = -20
+				is_at_war = yes
+			}
+		}
+	}
+	
+	#Coward
+	option = {
+		name = cen_kal.1.b
+		add_prestige = -35
+		if = { limit = { has_trait = craven } add_stress = -25 }
+		if = { limit = { has_trait = brave } add_stress = 35 }
+		scope:companion = {
+			add_opinion = {
+				modifier = weak_opinion
+				opinion = -15
+				target = root
+			}
+			add_dread = -3
+			trigger_event = { id = cen_kal.1015 days = { 2 5 } }
+		}
+		custom_tooltip = cen_kal.1.b.tt
+		ai_chance = { 
+			base = 50
+			modifier = {
+				add = 25
+				has_trait = craven
+			}
+			modifier = {
+				add = 25
+				has_trait = wounded_1
+			}
+			modifier = {
+				add = 20
+				is_at_war = yes
+			}
+		}
+	}
+	
+	#Hate other ruler
+	option = {
+		name = cen_kal.1.c
+		add_prestige = -35
+		scope:companion = {
+			add_opinion = {
+				modifier = weak_opinion
+				opinion = -10
+				target = root
+			}
+		}
+		scope:kodo_owner = {
+			add_opinion = {
+				modifier = weak_opinion
+				opinion = -20
+				target = root
+			}
+		}
+		ai_chance = { 
+			base = 50
+			modifier = {
+				add = 15
+				has_trait = craven
+			}
+			modifier = {
+				add = 15
+				has_trait = wounded_1
+			}
+			modifier = {
+				add = 20
+				is_at_war = yes
+			}
+			opinion_modifier = {
+				opinion_target = scope:kodo_owner
+				multiplier = -0.75
+			}
+		}
+	}
+}
+############################################################
+#Goblin Prospectors Throneroom
+cen_kal.1001 = {
+	type = character_event
+	title = cen_kal.1001_name
+	desc = cen_kal.1001.desc
+	theme = stewardship
+	override_background = throne_room
+	
+	cooldown = { years = 4 }
+	
+	left_portrait = {
+ 		character = ROOT
+	}
+	right_portrait = {
+		character = var:goblin_prospector
+		animation = steward
+	}
+	
+	trigger = {
+		NOR = { 
+			has_character_flag = has_working_depot
+			has_character_flag = depot_under_constr
+			root.primary_title = { is_mercenary_company = yes }
+			government_has_flag = government_is_theocracy
+		}
+		is_landed = yes
+		highest_held_title_tier >= tier_duchy
+	}
+	
+	immediate = {
+		create_character = {
+			location = root.location
+			trigger_race_giving_no_gene_effect = yes   
```

Comment:
```suggestion
```
you already gave race trait

Reply:
Without this line, the race will not change genes. Ie. goblins will still look like humans